components/workspacekit/cmd/rings.go

```diff
@@ -261,21 +267,21 @@ var ring1Cmd = &cobra.Command{
 			}
 		}
 
```

Comment:
In general the (very) methods in this file would benefit from a few, short comments to give some structure. In this case maybe:
`// starting ring2`

Reply:
I've added a few comments where I think they'd help.

In general I like to avoid comments that just say **what** the code is doing - that should be readable from the code and tends to get outdated quickly. Commenting how or why we're doing it, that's super valuable.